Delivery/Gootkit Malware

```diff
@@ -0,0 +1,13 @@
+// Surface suspicious create file events related to malicious download URLs
+let CF = DeviceFileEvents
+// Check for pattern match used by attacker
+| where FileOriginUrl matches regex @"down.php\?(?:id=|\w{6,8}=\w{5,7}&id=)(?:\w+&\w{6,7}=\w{5}|\w{112,})"
+| summarize by  FolderPath, FileSize, SHA256, DeviceId, Timestamp, InitiatingProcessFileName, FileOriginUrl, FileOriginReferrerUrl;
+////////
+// Below section is to surface active follow-on Command and Control as a result of the above behavior. Comment out the below join to see
+// only file create events where the malware may be present but has not yet been executed.
+////
```

Comment:
// Next section surfaces active follow-on Command and Control as a result of the preceding behavior. Comment out the join on line 10 
// to view only those file create events where the malware may be present but not yet been executed.

Reply:
@endisphotic  @martyav can you please upload the query in this template ? https://github.com/microsoft/Microsoft-365-Defender-Hunting-Queries/blob/master/00-query-submission-template.md

Reply:
Commit #4c6548f  added a template to the page